src/Agoda.Analyzers/AgodaCustom/CustomRulesResources.resx

```diff
@@ -249,6 +249,6 @@ One exception is logging, where it can be useful to see the exact DC / cluster /
     <value>Do not use #region directives</value>
   </data>
   <data name="AG0039Title" xml:space="preserve">
-    <value>Libraries with Public members must have documentation</value>
+    <value>Methods with more than 120 lines of code are potentially not readable, consider refactoring for better readability.</value>
```

Comment:
This also mentions 120 lines

Reply:
also removed, change to a parameter to the actual size of the method for now